specification/network/resource-manager/Microsoft.Network/preview/2022-04-01-preview/examples/NetworkManagerConnectionManagementGroupDelete.json

```diff
@@ -3,7 +3,8 @@
   "parameters": {
     "api-version": "2022-04-01-preview",
     "managementGroupId": "managementGroupA",
-    "networkManagerConnectionName": "TestNMConnection"
+    "networkManagerConnectionName": "TestNMConnection",
+    "force": false
```

Comment:
why did we add a force flag here? This resource has no children and is not involved with deployments